Jenkinsfile

```diff
@@ -15,78 +15,117 @@
-        env.ISOLATION_ID = sh(returnStdout: true, script: 'printf $BUILD_TAG | sha256sum | cut -c1-64').trim()
-        env.COMPOSE_PROJECT_NAME = sh(returnStdout: true, script: 'printf $BUILD_TAG | sha256sum | cut -c1-64').trim()
+        stage('Fetch Tags') {
+            steps {
+                sh 'git fetch --tag'
+            }
+		}
 
-        // Build the test dependencies
-        stage("Build test dependencies") {
-            sh 'docker-compose -f docker/compose/java-build.yaml build'
-            sh 'docker-compose -f docker/compose/java-build.yaml up'
-            sh 'docker-compose -f docker/compose/java-build.yaml down'
+        stage('Build Test Dependencies') {
+			steps {
+				sh 'docker-compose -f docker/compose/java-build.yaml build'
+				sh 'docker-compose -f docker/compose/java-build.yaml up'
+			}
         }
 
-        // Run the tests
-        stage("Run Tests") {
-            sh 'docker-compose -f examples/intkey_java/tests/test_intkey_smoke_java.yaml up --abort-on-container-exit'
-            sh 'docker-compose -f examples/intkey_java/tests/test_tp_intkey_java.yaml up --abort-on-container-exit'
-            sh 'docker-compose -f examples/xo_java/tests/test_tp_xo_java.yaml up --abort-on-container-exit'
-            sh 'docker-compose -f examples/xo_java/tests/test_xo_smoke_java.yaml up --abort-on-container-exit'
+        stage('Run Tests') {
+			steps {
+				sh 'docker-compose -f examples/intkey_java/tests/test_intkey_smoke_java.yaml up --abort-on-container-exit'
+				sh 'docker-compose -f examples/intkey_java/tests/test_tp_intkey_java.yaml up --abort-on-container-exit'
+				sh 'docker-compose -f examples/xo_java/tests/test_tp_xo_java.yaml up --abort-on-container-exit'
+				sh 'docker-compose -f examples/xo_java/tests/test_xo_smoke_java.yaml up --abort-on-container-exit'
+			}
         }
 
-        stage("Create git archive") {
-            sh '''
-                REPO=$(git remote show -n origin | grep Fetch | awk -F'[/.]' '{print $6}')
-                VERSION=`git describe --dirty`
-                git archive HEAD --format=zip -9 --output=$REPO-$VERSION.zip
-                git archive HEAD --format=tgz -9 --output=$REPO-$VERSION.tgz
-            '''
+        stage('Create Git Archive') {
+			steps {
+				sh '''
+					REPO=$(git remote show -n origin | grep Fetch | awk -F'[/.]' '{print $6}')
+					VERSION=`git describe --dirty`
+					git archive HEAD --format=zip -9 --output=$REPO-$VERSION.zip
+					git archive HEAD --format=tgz -9 --output=$REPO-$VERSION.tgz
+				'''
+			}
         }
 
-        stage("Archive Build artifacts") {
-            archiveArtifacts artifacts: '*.tgz, *.zip'
+	}
+
+    post {
+        always {
+            sh 'docker-compose -f docker/compose/java-build.yaml down'
+            sh 'docker-compose -f examples/intkey_java/tests/test_intkey_smoke_java.yaml down'
+			sh 'docker-compose -f examples/intkey_java/tests/test_tp_intkey_java.yaml down'
```

Comment:
Is there a reason for this indention?

Reply:
no, there were some tabs left over causing things to not line up correctly. Good catch, things looked normal on my system.